lang/src/test/kotlin/org/partiql/lang/eval/EvaluatingCompilerOrderByTests.kt

```diff
@@ -193,7 +193,7 @@ class EvaluatingCompilerOrderByTests : EvaluatorTestBase() {
             // text types compared by lexicographical ordering of Unicode scalar (ASC)
             EvaluatorTestCase(
                 """SELECT * FROM [{ 'a': `'\uD83D\uDCA9'`}, { 'a': 'Z'}, { 'a': '9' }, { 'a': 'A'}, { 'a': `"\U0001F4A9"`}, { 'a': 'a'}, { 'a': 'z'}, { 'a': '0' }] ORDER BY a""",
-                """[{'a': '0'}, {'a': '9'}, {'a': 'A'}, {'a': 'Z'}, {'a': 'a'}, {'a': 'z'}, {'a': `"\U0001F4A9"`}, {'a': `'\uD83D\uDCA9'`}]"""
+                """[{'a': '0'}, {'a': '9'}, {'a': 'A'}, {'a': 'Z'}, {'a': 'a'}, {'a': 'z'}, {'a': `'\uD83D\uDCA9'`}, {'a': `"\U0001F4A9"`}]"""
```

Comment:
The order of 2 values needs to be swapped, since it was not right originally. But originally, we did not detect it, since for `ExprValue.exprEquals()`, the 2 values are considered as equal.